src/main/js/components/subcomponents/DiningSession.js

```diff
@@ -23,4 +35,109 @@ export class CustomerDiningSessionSelect extends React.Component {
             </select>
         );
     }
+}
+
+export class StaffDiningSessionPage extends React.Component{
+
+    constructor(props) {
+        super(props);
+        this.state = {
+            show : false,
+            orders : []
+        }
+    }
+
+    render(){
+        const sessions = this.props.diningSessions.map(session =>
+            <StaffDiningSession
+                key={session.entity._links.self.href}
+                diningSession={session}
+                diningSessionAttributes={this.props.diningSessionAttributes}
+                history = {this.props.history}
+                onCreate={this.onCreate}
```

Comment:
onCreate does not need to be passed.
onUpdate/onDelete are not implemented throughout